### PR TITLE
Add a script to test installation script

### PR DESCRIPTION
**Pull request by Simon-Li**
For #497

### CODE REVIEW

File: scripts/install_test.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+fi
+
+echo "1. Check whether $TMP_DIR folder is removed.\n"
+if [ -d "$TMP_DIR" ]; then
+  echo "$TMP_DIR folder is not removed."
+  exit 1
+fi
+echo "passed\n"
+
+echo "2. Check whether $KUBEBUILDER_DIR folder is existed.\n"
```
Comment (droot) [inline scripts/install_test.sh]: s/is existed/exists

File: scripts/install_test.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+if [ ! -x "$KUBEBUILDER_DIR/bin/kubebuilder" ]; then
+  echo "$KUBEBUILDER_DIR/bin/kubebuilder is not existed or execute permission not granted."
+  exit 1
+fi
+echo "passed\n"
+
+echo "4. Check whether kubebuilder version is same as installed.\n"
+KUBEBUILDER_VERSION_FROM_BIN=$($KUBEBUILDER_DIR/bin/kubebuilder version | \
+      sed 's/^.*KubeBuilderVersion:"\(.*\)", KubernetesVendor.*$/\1/')
+if [ ! "${KUBEBUILDER_VERSION_FROM_BIN}" = "${KUBEBUILDER_VERSION}" ]; then
```
Comment (droot) [inline scripts/install_test.sh]: Nice :)

**Comment by droot** [comment]
Also, please squash all your commits into one commit.

File: scripts/install_test.sh (hunk 1 above)
Comment (Simon-Li) [reply scripts/install_test.sh]: done

**Comment by Simon-Li** [comment]
@droot updates pushed.

**Comment by droot** [comment]
@Simon-Li looks like the install test script is not getting executed in travis. Can you pl. investigate. Thanks

**Pull request merged**